### PR TITLE
First Version

### PR DESCRIPTION
**Pull request by alerizzo**

### CODE REVIEW

File: README 2.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Codacy ESLint
```
Comment (albertoadami) [inline README 2.md]: Just a minor thing, could we merge the README2 with README instead of having both files?

Comment (alerizzo) [reply README 2.md]: That's the idea, but for now i have no idea what to keep. This commit is full of things like this, just fyi, it's a temp first draft

Comment (albertoadami) [reply README 2.md]: I've merged into one single file, please check if it sounds good to you. I removed things related to ESlint that are not required for this plugin I think.

**Review by codacy-vrhpires — APPROVED** [review]
LGTM! 🚀

**Review by albertoadami — APPROVED** [review]

**Pull request merged**